html_css/intermediate_html/svgs.md

```diff
@@ -30,7 +30,7 @@ With vector graphics on the other hand, there's no grid. Instead, you have formu
 
 SVGs have another interesting aspect to them: they're defined using XML. XML (aka, "Extensible Markup Language") is an HTML-like syntax which is used for lots of things, from [APIs](https://en.wikipedia.org/wiki/SOAP), to [RSS](https://en.wikipedia.org/wiki/RSS), to [spreadsheet and word editor software](https://en.wikipedia.org/wiki/Office_Open_XML).
 
-The fact that SVG source-code is XML has a few key benefits.
+The fact that SVG source-code is XML which has a few key benefits.
```

Comment:
The original is correct here. The word "has" is a verb for which "fact" is the noun, not "XML".